llumnix/config/default.py

```diff
@@ -27,7 +27,7 @@
 # Port number for the server
 _C.SERVER.PORT = 8000
 # Queue type for request output queue
-_C.SERVER.QUEUE_TYPE = "rayqueue"
+_C.SERVER.QUEUE_TYPE = "zmq"
```

Comment:
keep rayqueue as default as zmq is not stable

Reply:
Done.